docs/manager/user-api/intro.rst

```diff
@@ -123,4 +123,20 @@ If you are implementing a web-based front-end, you may use it directly to the st
 When the item type is ``"log"``, the item data is a 4-tuple of the log level, the timestamp in the ISO 8601 format, the logger name and the log message string.
 The log level may be one of ``"debug"``, ``"info"``, ``"warning"``, ``"error"``, or ``"fatal"``.
 You may use different colors/formatting by the log level when printing the log message.
-Not every kernel runtime supports this rich logging facility.
\ No newline at end of file
+Not every kernel runtime supports this rich logging facility.
+
+.. _writing-new-api:
+
+Writing new REST API
+---------------
+Be advised that starting from 24.03, every new and updated REST APIs should adapt pydantic as its request and response validator. For starters our `service` API implementations can be a good boilerplate.
+
+.. note::
+   Do not adapt legacy trafaret-based approach for fresh new REST APIs! This approach is deprecated.
+
+Use `ai.backend.manager.api.utils.pydantic_response_api_handler` as a function decorator for API handlers without request body or queryparam to consume. Otherwise adapt `ai.backend.manager.utils.pydantic_params_api_handler`.
```

Comment:
Since this is a RestructuredText format, we should use double-backticks to format as inline codes. Single backticks mean that the text is given a "role" which can be extended by additional text processors. e.g., `` :func:`my_function()` ``. Single backtick without the role name is treated as "default role", which renders as a plain text unless explicitly configured.